Makefile.am

```diff
@@ -65,29 +65,19 @@ endif
 @DX_RULES@
 MOSTLYCLEANFILES += $(DX_CLEANFILES)
 
-### Library ###
-lib_LTLIBRARIES += libtpm2-totp.la
-include_HEADERS += include/tpm2-totp.h
-
-libtpm2_totp_la_SOURCES = src/libtpm2-totp.c
-libtpm2_totp_la_LIBADD = $(AM_LDADD)
-libtpm2_totp_la_LDFLAGS = $(AM_LDFLAGS) '(tpm2_totp)'
-
-pkgconfig_DATA = dist/tpm2-totp.pc
```

Comment:
`libtpm2-totp.so` is supposed to be not only an internal library, but something external applications (GUIs, system management applications, ...) can link against. Please do not remove it completely.